### DISCUSSION TITLE
So, there is a new Fluent design

### DISCUSSION THREAD

**Discussion by ReneeGA2020** [microsoft/microsoft-ui-xaml#10032] [General]
I don't know. Does that mean WinUI 3 will have a new set of style? Or that is a design we will never see in any microsoft product?
https://x.com/MicrosoftDesign/status/1836834503526076594

**Comment by mdtauk** [comment]
Its an evolution of the Illustration designs for Fluent.  Not a new design system.

**Comment by nikolayvpavlov** [comment]
Forget about Fluent. Microsoft still write their apps in WebView2, i.e. web apps, and the design is bleak.  Just look at the new Outlook and don't look any further.  Ugly beyond hope.

**Reply by mdtauk** [reply]
Fluent Design is used whether its WinUI or Fluent Web, its a shared design language
